website/blog/2023-07-12-release-1.2.md

```diff
@@ -0,0 +1,96 @@
+- **Kubernetes context on the status bar**: Kube context can now appear on the status bar
+- **Added start/stop/delete buttons for Compose**: You can now make changes to an entire Compose group of containers
+- **Ability to rename an image**: Rename an image with a click of a button
+
+Podman Desktop 1.2 is now available. [Click here to download it](/downloads)!
+
+![Podman-desktop-1-2-hero](img/podman-desktop-release-1.2/podman-desktop-release-1.2.png)
+
+---
+
+## Release Details
+
+### Protocol handler support
+
+Podman Desktop now supports protocol handling when using the terminal! If you type in `open podman-desktop:extension/redhat.openshift-local`, Podman Desktop will automatically load up to the correct extension.
+
+<ReactPlayer playing controls url="https://user-images.githubusercontent.com/436777/243304511-b11ad1e4-4c2f-455c-957a-01653d2a93c8.mp4" />
+
+### Kubernetes context on the status bar
+
+On the status bar, the current Kubernetes context is now shown. If there are multiple contexts, you can now pick which one to use.
+
+<ReactPlayer playing controls url="https://user-images.githubusercontent.com/19958075/243804525-242b02b4-fc3c-415b-be08-24eb1933adc5.mov" />
+
+### Added start/stop/delete buttons for Compose
+
+You can now start, stop and delete a group of containers launched by either `docker-compose` or `podman-compose`.
+
+<ReactPlayer playing controls url="https://user-images.githubusercontent.com/6422176/251518881-488d95fd-f5fe-4407-9656-f2fa83d8a592.mov" />
+<ReactPlayer playing controls url="https://user-images.githubusercontent.com/6422176/251522086-289177d7-176b-4fa0-bdcf-6b5cc5e1fd52.mov" />
+
+### Ability to rename an image
+
+Thanks to an awesome outside contributor [@tuckerrc](https://github.com/tuckerrc) you can now rename a container image.
+
+<ReactPlayer playing controls url="https://user-images.githubusercontent.com/436777/251759557-bd15a631-93ee-4383-a81c-8ef3934dfb59.mp4" />
+
+---
+
+## Other Notable Enhancements
+
+- Implemented dynamic breadcrumbs ([PR #3119](https://github.com/containers/podman-desktop/pull/3119))
+- Added icons on form pages ([PR #3155](https://github.com/containers/podman-desktop/pull/3155))
+- Introduced Reports warnings on failed kube deploy ([PR #3050](https://github.com/containers/podman-desktop/pull/3050))
+- Added troubleshooting guide for lima ([PR #2995](https://github.com/containers/podman-desktop/pull/2995))
+- Implemented background colors and FormPage ([PR #2977](https://github.com/containers/podman-desktop/pull/2977))
+- Switched to node16 modules instead of node modules in the main package ([PR #2788](https://github.com/containers/podman-desktop/pull/2788))
+- Introduced Minikube documentation ([PR #2694](https://github.com/containers/podman-desktop/pull/2694))
+- Added Minikube install docs ([PR #2824](https://github.com/containers/podman-desktop/pull/2824))
+- Enhanced the Troubleshooting page ([PR #2949](https://github.com/containers/podman-desktop/pull/2949))
+
```

Comment:
I don't see the improvment we've made when starting containers (ability to customize entrypoint and/or commands)